### PR TITLE
crio: remove DefaultsPath

### PR DESCRIPTION
**Pull request by kolyshkin**
Commit 1e54cdd362e6ff8e (PR #1740) added support for setting build-time DefaultsPath.

Commit 97b9587f160583d6 (PR #3321) added setting DefaultsPath from the Makefile.

Commit eb9cc161cb0f47b8 (PR #3298) renamed internal/criocli to internal/pkg/criocli.

The last two commits/PRs were merged the same day (5 March 2020).
This means the feature never worked (it could have worked in backports but they
never happened).

Since it never worked for 3+ years, it probably have no users.
    
Remove it.

Essentially, this reverts commits 1e54cdd362e6ff8e and 97b9587f160583d6.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by kolyshkin** [comment]
@jcvenegas @joselamego PTAL

**Comment by joselamego** [comment]
> @jcvenegas @joselamego PTAL

No objections from my side.

**Comment by haircommander** [comment]
wow good find
makes sense to me :+1: 
@cri-o/cri-o-maintainers PTAL

**Comment by umohnani8** [comment]
LGTM

**Comment by kolyshkin** [comment]
> I think we need a release note for this

Maybe we don't? Technically, we are not removing anything, since the feature never worked. It's like removing some dead code. Nothing changes from a user's perspective.

**Review by saschagrunert — APPROVED** [review]

**Comment by TomSweeneyRedHat** [comment]
LGTM once tests are hip

**Pull request merged**
